### PR TITLE
[FW] Enable Turrets By Default

### PR DESCRIPTION
**Pull request by DBmaoha**
Current FW mega turrets're diabled until they first seeing an enemy, make them unable to be damaged at start, which is confusing

### CODE REVIEW

**Review by GeckoEidechse — APPROVED** [review]
Confirmed working in testing.

Tested in 1-player listen server. Pre-PR enemy base turrets would need to "wake up" before shooting and taking damage. With PR applied they seem to be always active.

**Comment by GeckoEidechse** [comment]
Need someone to do code review and then it should be ready to merge ^^

**Review by Dinorush — APPROVED** [review]
Indentation on the pre-existing code could be adjusted to match the rest of the code, but otherwise the code itself looks fine to me.

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut [hunk 1]
```diff
@@ -1384,19 +1381,23 @@ void function FWAreaThreatLevelThink_Threaded()
...
+	thread FWTurretHighlightThink( turret )
 }
 
 // this will clear turret's highlight upon their death, for notifying players to fix them
 void function FWTurretHighlightThink( entity turret )
 {
-    turret.EndSignal( "OnDestroy" )
-    Highlight_SetFriendlyHighlight( turret, "fw_friendly" )
+	turret.EndSignal( "OnDestroy" )
+	WaitFrame()
```
Comment (EladNLG) [inline Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut:1396]: Please add a comment as to why is this here :)

**Review by EladNLG — APPROVED** [review]
With exceptions, looks good to me.

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut (hunk 1 above)
Comment (GeckoEidechse) [reply Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut:1396]: Once this addressed we should merge the PR ^^

**Review by EladNLG — CHANGES_REQUESTED** [review]
Accident :(

**Review by GeckoEidechse — APPROVED** [review]
Requested changes were addressed. Re-testing since last commit is not necessary as the only things changed are indentation and comments.
